MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,471 @@
+---
+layout: post
+title: Getting Started with Maui DataForm control | Syncfusion
+description: Learn here about getting started with Syncfusion Maui DataForm (SfDataForm) control, its elements and more.
+platform: Maui
+control: SfDataForm
+documentation: ug
+---
+
+# Getting Started with Maui DataForm (SfDataForm)
```

Comment:
# Getting Started with .NET MAUI  DataForm (SfDataForm)

Reply:
Maui to .NET MAUI